index.html

```diff
@@ -148,5 +147,22 @@ <h2>About Myself</h2>
             </div>
         </section>
     </main>
+    <footer class="contact">
```

Comment:
- Hey Team, great with the page layout and the form section implementation, however, this container should be a section tag and not a `footer` as this particular Figma design doesn't have a footer and as such, this should another section of your page. To improve our semantic usage, kindly update this to a section tag.

Reply:
- Changed the `footer` tag to a `section` tag for container and replaced the former `section` tag with a `div` tag.